src/app/shared/cms/components/cms-product-list-category/cms-product-list-category.component.ts

```diff
@@ -0,0 +1,44 @@
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { Observable } from 'rxjs';
+import { filter, map, switchMap, take } from 'rxjs/operators';
+
+import { ShoppingFacade } from 'ish-core/facades/shopping.facade';
+import { ContentPageletView } from 'ish-core/models/content-view/content-view.model';
+import { whenTruthy } from 'ish-core/utils/operators';
+import { CMSComponent } from 'ish-shared/cms/models/cms-component/cms-component.model';
+
+@Component({
+  selector: 'ish-cms-product-list-category',
+  templateUrl: './cms-product-list-category.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CMSProductListCategoryComponent implements CMSComponent, OnChanges {
+  @Input() pagelet: ContentPageletView;
+
+  productSKUs$: Observable<string[]>;
+
+  constructor(private shoppingFacade: ShoppingFacade) {}
+
+  ngOnChanges() {
+    if (this.pagelet.hasParam('Category')) {
+      this.productSKUs$ = this.getProductSKUs$();
+    }
+  }
+
+  getProductSKUs$(): Observable<string[]> {
+    return this.shoppingFacade.categoryIdByRefId$(this.pagelet.stringParam('Category')).pipe(
+      whenTruthy(),
+      switchMap(categoryId =>
+        this.shoppingFacade.productListingViewWithParams$(categoryId).pipe(
```

Comment:
The currently used action `loadMoreProductsForParams` triggers a not necessary `/productfilters` call as well. Maybe a more direct or more specific handling can be build/used that only fetches the category products and nothing else. Reusing the product list pages functionality might not be the best way since just fetching category products or just fetching filtered products seems a bit simpler.